### PR TITLE
add form for input and arrow

### PR DESCRIPTION
**Pull request by KaterynaMadzhara**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,33 +1,26 @@
 .App {
   text-align: center;
 }
-
+body {background-color: #282c34}
```
Comment (Romms) [inline src/App.css]: It could be in the same code style 
```suggestion
body {
  background-color: #282c34;
}
```

File: src/App.css [hunk 2]
```diff
@@ -46,6 +39,43 @@
 }
 button {
   border: none;
-  background-color: transparent;
+  background-color:transparent;
+  outline: none;}
```
Comment (Romms) [inline src/App.css]:
```suggestion
  outline: none;
}
```

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,17 @@
...
-            </button>
-            <button className='hover-button--on' onClick={()=> handleClick()}>
-                <img src={isActive? logo2: logo1}  className="App-logo" alt="logo"/>
-            </button>
-        </div>
-      </header>
-    </div>
+    <>
+        <Routes>
+            <Route path="/registered/:id" element={<RegisteredPage />}/>
```
Comment (Romms) [inline src/App.js]: you use 4 spaces or 2 spaces?
```suggestion
  <>
    <Routes>
      <Route path="/registered/:id" element={<RegisteredPage />}/>
```

File: src/components/RegisterUser.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React, {useState} from "react";
+
+import {Link}
+    from "react-router-dom";
```
Comment (Romms) [inline src/components/RegisterUser.js]:
```suggestion
import {Link} from "react-router-dom";
```

File: src/pages/registeredPage.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    from "react-router-dom";
+import background from '../img/confetee.png'
+
+function RegisteredPage() {
+    const {id} = useParams();
+
+    return (
+        <div style={{
+            backgroundImage: `url(${background})`,
+            backgroundPosition: 'center',
```
Comment (Romms) [inline src/pages/registeredPage.js]: can it be a css classname instead inlined styles?
```suggestion
        <div style={{
            backgroundImage: `url(${background})`,
            backgroundPosition: 'center',
```

File: src/components/RegisterUser.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+             <input placeholder="Імʼя" value={value} onChange={event => setValue(event.target.value)}/>
+             <Link to={`/registered/${value}`}>
+                 <button className='button-submit' type="submit">Зареєструватися</button>
+             </Link>
+
+         </form>
+         </>
+     )
+ }
+ export default RegisterUser
```
Comment (Romms) [inline src/components/RegisterUser.js]: some files have `export default` some `export { ... }` 
let's have one style

File: src/App.css (hunk 1 above)
Comment (KaterynaMadzhara) [reply src/App.css]: fixed

File: src/App.css (hunk 2 above)
Comment (KaterynaMadzhara) [reply src/App.css]: fixed

File: src/components/RegisterUser.js (hunk 4 above)
Comment (KaterynaMadzhara) [reply src/components/RegisterUser.js]: fixed

File: src/pages/registeredPage.js (hunk 5 above)
Comment (KaterynaMadzhara) [reply src/pages/registeredPage.js]: added classes

File: src/components/RegisterUser.js (hunk 6 above)
Comment (KaterynaMadzhara) [reply src/components/RegisterUser.js]: fixed

**Pull request merged**